.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,8 @@
 <!--
-  Please read the contributing guidelines before opening an issue:
+  ⚠ Please do not report outdated docs, we have an automated versions report:
```

Comment:
```suggestion
  ⚠ Please check the automated versions report before reporting outdated docs:
```
Since a numpy update was missed (https://github.com/freeCodeCamp/devdocs/issues/1352) how about this wording?

Reply:
Yes, your wording is better. :-)